packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -323,6 +323,7 @@ describe('internal-plugin-metrics', () => {
             showInterstitialTime: undefined,
             meetingInfoReqResp: undefined,
             registerWDMDeviceJMT: undefined,
+            getU2CTime: undefined,
```

Comment:
suggestion: This doesn't check the right method was used, simply that it was undefined (which could also be achieved by setting `joinTimes.getU2CTime = undefined`).